### ISSUE TITLE
Semantic / Latent

### ISSUE THREAD

**Issue by SophieZheng998** [HeYueThu/CausPref#1]
Hi! I am quite curious that does your model only works for datasets with semantic features? Since you are allowing new users/items in validation and testing datasets, the traditional way of learning latent features in matrix factorization might not work in this case?
Besides, do you plan to release your code recently?

**Comment by HeYueThu** [comment]
Yep! Our code will be released. Owing to our target OOD problem where the new user/item appears, we need the profile information of them. Thank you!

**Comment by HeYueThu** [comment]
Hi，I have uploaded a version on the GitHub https://github.com/HeYueThu/CausPref. If any problem, please contact me! Thank you for being interested in our work!
